### PR TITLE
feat(docs): copy across and refactor some docs from the /mojaloop repo

### PR DESCRIPTION
**Pull request by lewisdaly**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,81 @@
...
+## Contributing
+
+Refer to the [Contributors guide](https://docs.mojaloop.io/documentation/contributors-guide/) to get started making contributions.
+
+You may also want to read:
+- [Mojaloop Github Project](https://github.com/mojaloop/project)
+  > _note: You must install the [ZenHub extension](https://www.zenhub.com/extension) in order to see the Project Kanban Board_
+- [Developer Onboarding Guide](./onboarding.md)
+
+If you have any trouble getting started, or want to know where to start, reach out to us on the `#general` channel in [Slack](https://mojaloop-slack.herokuapp.com/)!
```
Comment (elnyry-sam-k) [inline README.md:39]:
```suggestion
If you have any trouble getting started, or want to know where to start, reach out to us on the `#help-mojaloop` channel in [Slack](https://mojaloop-slack.herokuapp.com/)!
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,81 @@
...
+Refer to the [Mojaloop Repository Overview](https://docs.mojaloop.io/documentation/repositories/) for a detailed breakdown of each Mojaloop Repository in development.
+
+As a quick reference, some of the most important repositories for contributors are:
+
+- [central-ledger](https://github.com/mojaloop/central-ledger) - Core Service in charge of managing transfers between DFSPs
+- [ml-api-adapter](https://github.com/mojaloop/ml-api-adapter) - Translation layer to convert to/from Mojaloop API to an internal format that is used in Central Services Stack.
+- [account-lookup-service](https://github.com/mojaloop/account-lookup-service) - A service for managing party accounts across a Mojaloop environment
+- [helm](https://github.com/mojaloop/helm) - The Helm chart development for deploying Mojaloop on Kubernetes
+- [postman](https://github.com/mojaloop/postman) - A set of Postman scripts for testing and managing a Mojaloop environment
+- [documentation](https://github.com/mojaloop/documentation) - The mojaloop documentation, published [here](https://docs.mojaloop.io/documentation)
```
Comment (elnyry-sam-k) [inline README.md:52]: Can you please add https://github.com/mojaloop/ml-testing-toolkit and https://github.com/mojaloop/project repositories as well :-)

File: onboarding.md [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+
+
+
+## <a name='Non-CLITools'></a>Non-CLI Tools
+
+1. [Github](#1-github)
+2. [Postman](#2-postman)
+3. [Zenhub](#3-zenhub)
+4. [Javascript IDE](#4-javascript-ide) (optional)
+5. [MySQLWorkbench](#5-mysqlworkbench) (optional)
```
Comment (elnyry-sam-k) [inline onboarding.md:204]: proposing TTK as we're not actively supporting postman (tests) moving forward

**Comment by elnyry-sam-k** [comment]
Looking good @lewisdaly - just some minor changes, especially to include ml-testing-toolkit (to replace postman )

File: onboarding.md (hunk 3 above)
Comment (lewisdaly) [reply onboarding.md:204]: Let's get rid of this file, It's covered in the docs already and is out of date

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**